lib/cli/options.ts

```diff
@@ -42,11 +41,11 @@ export class Option {
     if (value && typeof value === 'string') {
       return '' + value;
     }
-    return null;
+    return '';
```

Comment:
whether or not typescript thinks `null` is a string isn't the issue (though if it does, it's broken)--it's that you can't call `split()` on `null`.